src/actions/types.js

```diff
@@ -0,0 +1,3 @@
+export const SET_CONFIG = 'SET_CONFIG';
+export const SET_LANGUAGE = 'SET_LANGUAGE';
+export const THEME_COLOR = 'THEME_COLOR';
```

Comment:
If the other ones have `SET_` as a prefix, probably `THEME_COLOR` should be `SET_THEME_COLOR`, no?